src/contexts/UserResponseContext.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {
+  createContext,
+  Dispatch,
+  SetStateAction,
+  useState,
+} from 'react';
+
+interface UserResponseState {
+  userResponseState: {
+    items: {
+      id: string;
+      userID: string;
+      questionID: string;
+      answer: string;
+      createdAt: string;
+    }[];
+  };
+  setUserResponseState:
+    | Dispatch<
+        SetStateAction<{
+          items: {
+            id: string;
+            userID: string;
+            questionID: string;
+            answer: string;
+            createdAt: string;
+          }[];
+        }>
+      >
+    | (() => void);
+}
+
+// TODO: Copied over from ResponseContext so need to change json values
+const initialState: UserResponseState = {
+  userResponseState: {
+    items: [
+      {
+        id: '574af3cc-68d6-4051-8be2-9a6353edaeba',
+        userID: '8f3431bd62-a301-4f0b-8e52-af30d940b615',
+        questionID: '97e9d4eb-0614-4548-961c-1fa30d5c0fca',
+        createdAt: '2021-05-18T07:04:04.878Z',
+        answer: 'Yes',
+      },
```

Comment:
Remove these hard coded items, they were left over from testing and never got removed. So we should just have
```javascript
  userResponseState: {
    items: [],
  },
```